### PR TITLE
Upgrade note

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
Adding P1 feature of the note app.

### CODE REVIEW

File: src/Components/MainBody/SubComponent/MainSubComponent.jsx [hunk 1]
```diff
@@ -1,75 +1,45 @@
...
-			: filterNotes &&
-			  filterNotes.length > 0 &&
-			  state.sortBy != null &&
-			  state.sortBy === "Pinned"
-			? filterNotes.filter((item) => item.pinned === true)
-			: filterNotes.filter((item) => item.tags === state.sortBy);
+		state.categoryFilter.length !== 0
+			? getFilterCategory(filterNotesDate, state)
+			: getFilterLabel(filterNotesDate, state);
+	console.log("notes", StickyNotes);
```
Comment (Shubhamkr2610) [inline src/Components/MainBody/SubComponent/MainSubComponent.jsx]: suggestion: remove console.log

**Review by Shubhamkr2610 — APPROVED** [review]
appreciation : Great work @santhoshpemmaka . good naming with classes and your logic of implementing the feature is also great . code is readable and easy to understand . you can proceed for merge

**Comment by VarshithSSV** [comment]
Great work. code is more readable and easy to understand. 

Good to merge 👍 .

**Review by VarshithSSV — APPROVED** [review]

**Comment by santhoshpemmaka** [comment]
Add : P1 features to the notes app.

**Pull request merged**